doc/frameworks/pytorch/using_pytorch.rst

```diff
@@ -292,7 +292,104 @@ using two ``ml.p4d.24xlarge`` instances:
 
     pt_estimator.fit("s3://bucket/path/to/training/data")
 
+.. _distributed-pytorch-training-on-trainium:
+=============================================
```

Comment:
```suggestion

Distributed PyTorch Training on Trainium
========================================
```

Reply:
Reverted this change to earlier version. Can you check now?